--- antiLeech.cpp.win	2026-10-02 20:45:24.261233220 +0800
+++ antiLeech.cpp	2026-10-03 09:02:27.473739202 +0800
@@ -30,6 +30,8 @@
 #include <atlstr.h>
 #include "antiLeech.h"
 
+const DWORD CantiLeech::DLPVersion = 4406;
+
 #ifdef _DEBUG
 #define new DEBUG_NEW
 #endif
@@ -89,7 +91,7 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL,
 
 //SDC team start
 #if defined(SPECIAL_DLP_VERSION)
-const DWORD __declspec(dllexport) GetDLPVersion(void)
+DWORD __declspec(dllexport) GetDLPVersion(void)
 {
 	return DLPVERSION;
 }
@@ -319,7 +321,7 @@ LPCTSTR __declspec(dllexport) DLPCheckNa
 
 //SDC team start
 #if defined(SPECIAL_DLP_VERSION)
-const bool IsTypicalHex(const CString & addon)
+bool CantiLeech::IsTypicalHex(const CString & addon)
 #else //Official
 bool IsTypicalHex(CString & addon)
 #endif
@@ -1322,7 +1324,7 @@ LPCTSTR __declspec(dllexport) DLPCheckUs
 
 //SDC team start
 #if defined(SPECIAL_DLP_VERSION)
-LPCTSTR __declspec(dllexport) DLPCheckNameAndHashAndMod(const CString username, const CString & userhash, const CString & modversion)
+LPCTSTR __declspec(dllexport) DLPCheckNameAndHashAndMod(const CString & username, const CString & userhash, const CString & modversion)
 #else //Official
 LPCTSTR __declspec(dllexport) DLPCheckNameAndHashAndMod(CString username, CString & userhash, CString & modversion)
 #endif
@@ -1817,8 +1819,8 @@ LPCTSTR __declspec(dllexport) DLPCheckHe
 		case CT_UNKNOWNx76:
 		case CT_UNKNOWNxCD:			strSnafuTag = apszSnafuTag[16];break; //buffer = _T("www.donkey2002.to");break;
 		case CT_UNKNOWNx88:			strSnafuTag = apszSnafuTag[5];break; //[LSD7c]
-		case CT_UNKNOWNx8C:			strSnafuTag = apszSnafuTag[5];break; //buffer = _T("[LSD7c]");break;
-		case CT_UNKNOWNx8D:			strSnafuTag = apszSnafuTag[6];break; //buffer = _T("[0x8d] unknown Leecher - (client version:60)");break;
+		case CT_UNKNOWNx8c:			strSnafuTag = apszSnafuTag[5];break; //buffer = _T("[LSD7c]");break;
+		case CT_UNKNOWNx8d:			strSnafuTag = apszSnafuTag[6];break; //buffer = _T("[0x8d] unknown Leecher - (client version:60)");break;
 		case CT_UNKNOWNx99:			strSnafuTag = apszSnafuTag[7];break; //buffer = _T("[RAMMSTEIN]");break; //STRIKE BACK
 		case CT_UNKNOWNx97:
 		case CT_UNKNOWNx98:
@@ -1828,7 +1830,7 @@ LPCTSTR __declspec(dllexport) DLPCheckHe
 		case CT_UNKNOWNxCE:										//Xman 20.08.05
 		case CT_UNKNOWNxCF:										//Xman 20.08.05
 		case CT_UNKNOWNx94:										//Xman 20.08.05
-		case CT_UNKNOWNxC4:			strSnafuTag = apszSnafuTag[8];break; //buffer = _T("[MD5 Community]");break; //USED BY NEW BIONIC => 0x12 Sender
+		case CT_UNKNOWNxc4:			strSnafuTag = apszSnafuTag[8];break; //buffer = _T("[MD5 Community]");break; //USED BY NEW BIONIC => 0x12 Sender
 		case CT_UNKNOWNxEC:			strSnafuTag = apszSnafuTag[18];break; //Xman x4 Speedmule
 //		case CT_FRIENDSHARING:		break;						//STRIKE BACK
 		case CT_DARK:											//STRIKE BACK
